style.css

```diff
@@ -0,0 +1,768 @@
+  align-items: center;
+  gap: 25px;
+}
+
+.footer-text {
+  color: #d9d0bf;
+}
+
+.footer-link {
+  text-decoration: none;
+  color: #d9d0bf;
+}
+
+.footer-link:hover {
+  color: #ffffff;
+}
+
+.popup {
+  position: fixed;
+  width: 100%;
+  height: 100%;
+  background-color: rgb(0, 0, 0, 0.8);
+  top: 0;
+  left: 0;
+  opacity: 0;
+  visibility: hidden;
+  overflow-y: auto;
+  overflow-x: hidden;
+  transition: all 0.5s ease 0s;
+  font-family: Inter;
+}
+
+.popup:target {
+  opacity: 1;
+  visibility: visible;
+}
+
+.popup-area {
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  top: 0;
+  left: 0;
+}
+
+.popup-body {
+  min-height: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 30px, 10px;
+}
+
+.popup-content {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  gap: 20px;
+  background-color: #f2e9d8;
+  color: #000000;
+  max-width: 40%;
+  min-width: 35%;
+  padding: 30px;
+  position: relative;
+}
+
+.popup-close {
+  position: absolute;
+  right: 10px;
+  top: 10px;
+  font-size: 20px;
+  color: #000000;
+  text-decoration: none;
+}
+
+.popup-form {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+}
+
+.popup-input {
+  font-size: 20px;
+  width: 100%;
+  height: 25px;
+  font-family: Inter;
+  font-style: normal;
+  font-weight: normal;
+}
+
+.popup-button {
+  height: 35px;
+  width: fit-content;
+  padding: 0 20px;
+  font-family: Inter;
+  font-style: normal;
+  font-weight: normal;
+  background-color: #d9d0bf;
+  border: 2px solid #363940;
+  color: #363940;
+  font-size: 20px;
+}
+
+.popup-button:hover {
+  color: #ffffff;
+}
+
+.date {
+  display: flex;
+  justify-content: flex-start;
+  gap: 20px;
+}
+
+@media screen and (max-width: 575px) {
+  .mobile-menu {
+    display: block;
+  }
+
+  .menu {
+    display: none;
+  }
+
+  .aside {
+    display: none;
+  }
+
+  .mobile-aside-container {
+    position: fixed;
+    width: 100%;
+    height: 100%;
+    background-color: rgb(0, 0, 0, 0.8);
+    top: 0;
+    left: 0;
+    opacity: 0;
+    visibility: hidden;
+    overflow-y: auto;
+    overflow-x: hidden;
+  }
+
+  .mobile-aside-container-active {
+    position: fixed;
+    width: 100%;
+    height: 100%;
+    background-color: rgb(0, 0, 0, 0.8);
+    top: 70px;
+    bottom: 60px;
+    left: 0;
+    overflow-y: auto;
+    overflow-x: hidden;
+    opacity: 1;
+    visibility: visible;
+  }
+
+  .mobile-aside {
+    display: none;
+    padding-top: 10px;
+    top: 70px;
+    bottom: 60px;
+    position: fixed;
+    overflow-y: hidden;
+    overflow-x: hidden;
+    width: 60%;
+    right: 0;
+    border-left: 2px solid #363940;
+    transition: all 0.4s ease;
+    overflow: auto;
+  }
+
+  .mobile-aside-active {
+    display: block;
+  }
+
+  .footer-container {
+    justify-content: space-between;
+  }
+
+  .enter {
+    display: none;
+  }
+
+  .popup-content {
+    min-width: 80%;
+  }
+
+  .popup-button {
+    max-width: 50%;
+  }
```

Comment:
if I'm not mistaken, this could be removed ("Add theme" still broken on mobile screen)